docs/sphinx/releases.rst

```diff
@@ -4,10 +4,21 @@ CUDA Quantum Releases
 
 **latest**
 
-The latest version of CUDA Quantum is on the main branch of our `GitHub repository <https://github.com/NVIDIA/cuda-quantum>`__.
+The latest version of CUDA Quantum is on the main branch of our `GitHub repository <https://github.com/NVIDIA/cuda-quantum>`__, as well as available as Docker image. More information about installing the nightly builds can be found :doc:`here <install>`
 
+- `Docker image (nightly builds) <https://catalog.ngc.nvidia.com/orgs/nvidia/teams/nightly/containers/cuda-quantum>`__
 - `Documentation <https://nvidia.github.io/cuda-quantum/latest>`__
 
+**0.4.1**
+
+The 0.4.1 release adds support for ARM processors in the form of multi-platform Docker images and `aarch64` Python wheels. Additionally, all GPU-based backends are now included in the Python wheels as well as in the Docker image.
+
+- `Docker image <https://catalog.ngc.nvidia.com/orgs/nvidia/containers/cuda-quantum>`__
```

Comment:
This webpage has examples for pulling 0.3.0, so that page should probably be updated to give examples to pull 0.4.1 instead once it's available. (Looks like we just skipped 0.4.0 on that page.)